### DISCUSSION TITLE
What if I want to create a credit note with bollo a carico del cliente?

### DISCUSSION THREAD

**Discussion by Ouija1733** [fattureincloud/api#363] [Q&A] [answered]
As the title says I would like to know how to correctly create a credit note with bollo a carico del cliente and correctly send it to the Agenzia delle entrate.

Thank you in advence!

**Comment by emajo** [comment, marked as answer]
Hi @Ouija1733, we have an extensive guide on how to create an Issued Document, in particular in [this section](https://developers.fattureincloud.it/docs/guides/invoice-creation/#5%EF%B8%8F⃣-step-five-items-list) it is specified how to deal with the bollo, you can also find a [guide](https://developers.fattureincloud.it/docs/guides/e-invoice-management#-send-the-e-invoice) to send the doc to the Agenzia delle Entrate.

Thanks,
Emanuele.

**Reply by Ouija1733** [reply]
Hello @emajo, thanks for the fast reply!
So it doesen't change if the document is an Invoice or a Credit note?

**Reply by emajo** [reply]
No, the type changes but is still an issued document.
